plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/totals/shipping/index.tsx

```diff
@@ -27,20 +29,24 @@ export const TotalsShipping = ( {
 	collaterals = null,
 }: TotalShippingProps ): JSX.Element | null => {
 	const { cartTotals, shippingRates } = useStoreCart();
-	const hasRates = hasShippingRate( shippingRates );
+	const hasSelectedRates = hasSelectedShippingRate( shippingRates );
+	const rateNames = getSelectedShippingRateNames( shippingRates );
+	const hasMultipleRates = rateNames.length > 1;
+	const rowLabel =
+		! hasSelectedRates || hasMultipleRates ? label : rateNames[ 0 ];
```

Comment:
```suggestion
		( ! hasSelectedRates || hasMultipleRates ) ? label : rateNames[ 0 ];
```
A tiny suggestion, operator precedence is not yet engrained in my brain, even after years doing JS 😅